sirepo/package_data/static/js/cloudmc.js

```diff
@@ -3035,6 +3055,9 @@ SIREPO.viewLogic('tallySettingsView', function(appState, cloudmcService, panelSt
             'openmcAnimation.showSources',
             'openmcAnimation.numSampleSourceParticles',
         ], showFields,
+        ['openmcAnimation.score'], preserveScore,
+        ['openmcAnimation.tally'], useCachedScore,
```

Comment:
We are already watching  `openmcAnimation.tally` so it's better to cache the score as part of `validateTally()`.